monai/transforms/post/dictionary.py

```diff
@@ -500,10 +508,26 @@ def __init__(
 
     def __call__(self, data: Mapping[Hashable, Any]) -> Dict[Hashable, Any]:
         d = dict(data)
-        for key, orig_key, nearest_interp, to_tensor, device, post_func in self.key_iterator(
-            d, self.orig_keys, self.nearest_interp, self.to_tensor, self.device, self.post_func
+        for (
+            key,
+            orig_key,
+            meta_key,
+            meta_key_postfix,
+            nearest_interp,
+            to_tensor,
+            device,
+            post_func,
+        ) in self.key_iterator(
+            d,
+            self.orig_keys,
+            self.meta_keys,
+            self.meta_key_postfix,
+            self.nearest_interp,
+            self.to_tensor,
+            self.device,
+            self.post_func,
```

Comment:
For legibility it might help to put the arguments to `key_iterator` into a tuple on the line above and expand it here so that the `for` statement is on one line.

Reply:
Hi @ericspod ,

Thanks for your suggestion, I tried to use:
```py
iterators = (d, self.orig_keys, self.meta_keys, self.meta_key_postfix, self.nearest_interp, self.to_tensor, self.device, self.post_func)
for (
    key, orig_key, meta_key, meta_key_postfix, nearest_interp, to_tensor, device, post_func
) in self.key_iterator(iterators):
```
The first line is still more than 120 chars, maybe I misunderstood your idea?

Thanks.

Reply:
So now I just changed to below format in the latest commit:
```py
for (
    key, orig_key, meta_key, meta_key_postfix, nearest_interp, to_tensor, device, post_func
) in self.key_iterator(
    d,
    self.orig_keys,
    self.meta_keys,
    self.meta_key_postfix,
    self.nearest_interp,
    self.to_tensor,
    self.device,
    self.post_func,
):
```
What do you think?

Reply:
It's better but something like this might be more readable:

```python
iterators = (d, self.orig_keys, self.meta_keys, self.meta_key_postfix, self.nearest_interp, self.to_tensor, self.device, self.post_func)
for vars in self.key_iterator(*iterators):
    key, orig_key, meta_key, meta_key_postfix, nearest_interp, to_tensor, device, post_func = vars
```

Having the `for` on one line might be easier to read, up to you if you want to change it.

Reply:
I see, but the problem is that below line of your example code is more than 120 chars:
```py
iterators = (d, self.orig_keys, self.meta_keys, self.meta_key_postfix, self.nearest_interp, self.to_tensor, self.device, self.post_func)
```
So we still need to break it into lines...

Thanks.